pkgs/development/interpreters/python/default.nix

```diff
@@ -99,7 +99,8 @@ with pkgs;
         inherit sourceVersion;
         pythonAtLeast = lib.versionAtLeast pythonVersion;
         pythonOlder = lib.versionOlder pythonVersion;
-        inherit hasDistutilsCxxPatch pythonForBuild;
+        inherit hasDistutilsCxxPatch;
+        pythonForBuild = pythonOnBuildForHost;
```

Comment:
Maybe we add a comment here saying the `ForXx` mismatch is intentional and should be fixed by renaming `pythonForBuild`?